frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -54,6 +61,46 @@ function ReaderCoptListener:setAndSave(setting, property, value)
     UIManager:broadcastEvent(Event:new("SetStatusLine", self.document.configurable.status_line, true))
 end
 
+function ReaderCoptListener:UpdateHeader()
+    if self.view.view_mode == "page" then
+        if self.ui.document.provider == "crengine" then -- can that "if" be dropped
+            self.ui.document._callCacheSet("current_buffer_tag", nil)
+        end
+        self.ui:handleEvent(Event:new("RedrawCurrentView", self.current_page))
+    end
+end
+
+function ReaderCoptListener:setupAutoRefreshTime()
+    if not self.autoRefreshTime then
+        self.autoRefreshTime = function()
```

Comment:
Best to rename all that to AutoRefreshHeader - so there's no confusion with setting the refresh time/duration :)

Reply:
Yes, of course.